plugin/remote/github/helper.go

```diff
@@ -140,8 +140,21 @@ func GetOrgRepos(client *github.Client, org string) ([]github.Repository, error)
 // GetOrgs is a helper function that returns a list of
 // all orgs that a user belongs to.
 func GetOrgs(client *github.Client) ([]github.Organization, error) {
-	orgs, _, err := client.Organizations.List("", nil)
-	return orgs, err
+	var allOrgs []github.Organization
+	var err error
+	page := 1
```

Comment:
There are a few [places in the code](https://github.com/udzura/drone/blob/master/plugin/remote/github/helper.go#L95) where we loop through the paginated lists. Could we slightly adjust the function so that it is consistent with the other loops? Thanks!

```
	var orgs []github.Organization
	var opts = github.ListOptions{}
	opts.PerPage = 100
	opts.Page = 1

	// loop through user org list
	for opts.Page > 0 {
		list, resp, err := client.Organizations.List("", options)
		if err != nil {
			return nil, err
		}
		orgs = append(orgs, list...)

		// increment the next page to retrieve
		opts.Page = resp.NextPage
	}
```